po/pt_BR.po

```diff
@@ -36,36 +36,35 @@ msgstr "Personalizar a Aparência e o Comportamento"
 
 #: ../data/lxappearance.desktop.in.h:2
 msgid "Customizes look and feel of your desktop and applications"
-msgstr ""
-"Configurações da aparência e do comportamento da área de trabalho e dos "
-"programas aplicativos com o LXAppearance"
+msgstr "O ‘LXAppearance’ permite alterar as configurações da "
```

Comment:
This is the tooltip for a menu item. Please translate without using the application's name.

Reply:
> This is the tooltip for a menu item. Please translate without using the application's name.

Hello, ib.

In order for me to answer your question, I need to comment on the basic structure of the Portuguese language (pt_BR).

Sentences in Portuguese must have a subject, verb and predicate, in addition to having a beginning, middle and end. Sentences must make complete sense to native speakers of the respective language. For people who only understand their native language, when I mention the name of the program that is written in English, it also serves to reinforce and fix the writing of something that is foreign to their language and that people are not able to understand. read and understand what it is and what it is for just by reading the name "LXAppearance".

In the case of the phrase "#: ../data/lxappearance.desktop.in.h:2", it will be used in the "Comment[pt_BR]=" entry of the "lxappearance.desktop" file. Therefore, the subject of the sentence is "LXAppearance", the verb is the actions that the program has the ability to perform and in the rest of the sentence we have nouns, adjectives, prepositions, etc. I realize that in English it is customary for people to omit the subject and verb in the sentence, but in my language the subject and verb are the most important elements in the sentence. Automatic translation services on the internet are not capable of including the subject and verb in the sentence, and it is up to the human being to identify the appropriate verb that can be included in the translation and which is the subject in the sentence.

It should be noted that I am a voluntary translator of several GNU/Linux programs, you can check this information on my pages on the platforms I have a user account on, including: https://github.com/marcelocripe, https://gitlab .com/marcelocripe, LXQt Weblate, GitLab Salsa, SourceForge, Codeberg, etc (all with the username "marcelocripe"). The translation model that I used in "LXAppearance" is the same model that I used in several other ".desktop" files that you can check out in the project https://gitlab.com/Robin-antiX/antix23-desktop-files/-/tree/main/applications?ref_type=heads.

Launcher-type programs, such as Rofi, AppSelect (from antiX), Albert, Kupfer, Ulauncher, etc. display the entry text "Name[pt_BR]=Customize Appearance and Behavior" and the entry text "Comment[pt_BR]=The 'LXAppearance' allows you to change the appearance and behavior settings of the desktop and programs", where was the original name of the program displayed? Yes, the original program name information was displayed in the "Comment[pt_BR]=" entry for the end user.

Please accept my request to merge the most recent translation revision file that I sent you, with these revisions and improvements, the texts will be much easier to understand by Brazilians.

I would like to thank you and all the people who are involved in the "LXAppearance" project, I have been using this program on antiX Linux for over 4 years.

Thank you very much.

- - - - -

> This is the tooltip for a menu item. Please translate without using the application's name.

Olá, ib.

Para eu poder responder ao seu questionamento, eu preciso comentar sobre a estrutura básica da língua portuguesa (pt_BR).

As frases em idioma português precisam possuir o sujeito, o verbo e o predicado, além de que, tem que ter o início, o meio e o fim. As frases precisam fazer sentido completo para os nativos do respectivo idioma. Para as pessoas que só compreendem o seu idioma nativo, quando eu menciono o nome do programa que está escrito em idioma inglês, também serve para reforçar e fixar a escrita de algo que é estranho ao seu idioma e de que as pessoas não são capazes de ler e entender o que é e para que serve apenas lendo o nome "LXAppearance".

No caso da frase "#: ../data/lxappearance.desktop.in.h:2", será utilizada na entrada "Comment[pt_BR]=" do arquivo "lxappearance.desktop". Por tanto, o sujeito da frase é o "LXAppearance", o verbo são as ações que o programa possui a capacidade de realizar e no restante da frase temos os substantivos, os adjetivos, as preposições, etc. Eu percebo que em idioma inglês é costume das pessoas omitirem o sujeito e o verbo na frase, mas no meu idioma o sujeito e o verbo são os elementos mais importantes na frase. Os serviços de tradução automática da internet não são capazes de incluir o sujeito e o verbo na frase, cabendo ao ser humano identificar o verbo adequado que pode ser incluído na tradução e qual é o sujeito na frase.

Cabe salientar que sou um tradutor voluntário de vários programas do GNU/Linux, você pode conferir estas informações nas minhas páginas das plataformas que possuo uma conta de usuário, entre elas: https://github.com/marcelocripe, https://gitlab.com/marcelocripe, LXQt Weblate, GitLab Salsa, SourceForge, Codeberg, etc (todos com o nome de usuário "marcelocripe"). O modelo da tradução que eu utilizei no "LXAppearance" é o mesmo modelo que eu utilizei em vários outros aquivos ".desktop" que você pode conferir no projeto https://gitlab.com/Robin-antiX/antix23-desktop-files/-/tree/main/applications?ref_type=heads.
Os programas do tipo lançadores, como por exemplo, o Rofi, AppSelect (do antiX), Albert, Kupfer, Ulauncher, etc.exibem o texto da entrada "Name[pt_BR]=Personalizar a Aparência e o Comportamento" e o texto da entrada "Comment[pt_BR]=O ‘LXAppearance’ permite alterar as configurações da aparência e do comportamento da área de trabalho e dos programas", em qual lugar o nome original do programa foi exibido? Sim, a informação do nome original do programa foi exibido na entrada "Comment[pt_BR]=" para o usuário final.

Por favor, aceite o meu pedido de mesclagem do arquivo da revisão da tradução mais recente que eu te enviei, com estas revisões e melhorias, os textos serão muito mais fáceis de serem compreendidas pelos brasileiros.

Eu gostaria de agradecer a você e a todas as pessoas que estão envolvidas no projeto do "LXAppearance", eu utilizo este programa no antiX Linux há mais de 4 anos.

Muito obrigado.